### DISCUSSION TITLE
How to enable recording

### DISCUSSION THREAD

**Discussion by Yota73** [edumeet/edumeet-client#176] [Q&A]
Hello, I see that the recording is disabled by default. There is a way to enable it or it needs to be implemented ?
Thanks a lot

**Comment by havfo** [comment]
Seems recording was broken in a recent update to support P2P. Should now be back, and now also supports Firefox and Safari as well as Chrome.

**Reply by Yota73** [reply]
Thanks havfo, as soon as I figure out how to setup my local development without always recompile the docker images I'll try
